config/rooms-api.local.yaml

```diff
@@ -18,6 +18,12 @@ adapters:
   instanceStorage:
     redis:
       url: "redis://localhost:6379/0"
+  schedulerStorage:
```

Comment:
WDYT to reorganize the configurations in contexts? 
You could create scheduler context and put all scheduler configuration there.

would be like this:

```
scheduler:
       storage:
             postgres:
                   url: "postgres://user:pwd@localhost:5432/maestro?sslmode=disable"
        cache:
             redis:
                    url: "redis://localhost:6379/0"
```

Reply:
It's a change of pattern used now on next and will require some refactoring. Do you think this could be done on another PR?